3rdparty/hal_rvv/hal_rvv_1p0/norm.hpp

```diff
@@ -1002,6 +1002,7 @@ inline int norm(const uchar* src, size_t src_step, const uchar* mask, size_t mas
         },
     };
 
+    // [FIXME] append more when merging to 5.x
```

Comment:
I propose to add assert for `elem_size_tab[depth]` before usage. Just to highlight issue, if table is not complete or data type is unexpected.